app/uk/gov/hmrc/exports/controllers/testonly/TestOnlyController.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.controllers.testonly
+
+import play.api.libs.json.Json
+import play.api.mvc.{Action, ControllerComponents, Request}
+import uk.gov.hmrc.exports.controllers.RESTController
+import uk.gov.hmrc.exports.models.Eori
+import uk.gov.hmrc.exports.models.declaration._
+import uk.gov.hmrc.exports.models.declaration.notifications.{NotificationDetails, ParsedNotification}
+import uk.gov.hmrc.exports.models.declaration.submissions.{Submission, SubmissionRequest, Action => SubmissionAction}
+import uk.gov.hmrc.exports.models.declaration.submissions.SubmissionStatus.{ACCEPTED, ADDITIONAL_DOCUMENTS_REQUIRED, SubmissionStatus}
+import uk.gov.hmrc.exports.repositories.{DeclarationRepository, ParsedNotificationRepository, SubmissionRepository}
+import uk.gov.hmrc.exports.util.ExportsDeclarationBuilder
+
+import java.time.{LocalDateTime, ZoneId, ZonedDateTime}
+import java.util.UUID
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Random
+
+@Singleton
+class TestOnlyController @Inject()(
```

Comment:
I would give a specific name to this class, not `TestOnlyController`. This file is already quite big, and if in future we need to add an additional test endpoint, we do not want for sure to add its logic to this controller again.